### PR TITLE
UI of Home, Sign in and Sign up completed

### PR DESCRIPTION
**Pull request by rakesh-201**
A few thing I would like to clear:
- The theme of the site is a point of discussion and hence I have stored all the colors used in this site in a color.js file and importing from this file to all the components, so that the theme changing is a bit easy
- All the pages are responsive
- In the App.tsx file there is a variable temporary which when set to 1 we can see the site for logedin users and when set to 0 the site for logedout users appear.
- The NotesOverview Screen isn't complete because someone else is doing it.
- Also, I apologies for the delay, happened due to some genuine issue.

Some Images are attached below: 
Home
Home
Sign Up

### CODE REVIEW

**Comment by rakesh-201** [comment]
@AliabbasMerchant @ShubhankarKG

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,26 +1,13 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  const temporary = 0;
```
Comment (AliabbasMerchant) [inline client/src/App.tsx:8]: Please give better and descriptive names to variables

**Review by AliabbasMerchant — CHANGES_REQUESTED** [review]
1. Don't create 2 separate components for Desktop and Mobile. 1 Single component should be able to handle both. (pass a `mobile` boolean prop if-and-only-if needed)
2. Please remove the long, extra commented out stuff
3. Next time onwards, please do not make huge formatting changes in a PR that also has new/edited stuff to be reviewed. The PR size increases significantly. You can create a separate commit and then indicate that, or even better, make a separate PR itself.

**Comment by rakesh-201** [comment]
All the mentioned changes are done.

**Review by ShubhankarKG — APPROVED** [review]

**Pull request merged**